util/mbed/mbed-copy-osx.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
```

Comment:
Same comment about python versioning, if you can put a 2 or 3 after `python` to indicate which one it works with, it'll prevent version issues on different systems.